### PR TITLE
feat : add badge component

### PR DESCRIPTION
**Pull request by Kavita539**
Added Badge Component.
It has two types : Profile badge and Number Badge
You can browse preview on netlify deploy

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+.badge .offline-badge{
+    background-color: var(--grey);
+}
+
+.badge .dnd-badge{
+    background-color: var(--danger-color);
+}
+
+
```
Comment (prathmeshjagtap) [inline components/badge/badge.css]: you could directly give styling as  .online-badge{
background-color: var(--grey);
}

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+.badge .number{
```
Comment (prathmeshjagtap) [inline components/badge/badge.css]: you can be more specific with the number class name like badge-number.

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]
Added some suggestions everything else looks fine,

File: components/badge/badge.css (hunk 1 above)
Comment (Kavita539) [reply components/badge/badge.css]: ok will do the changes mentioned!

File: components/badge/badge.css (hunk 2 above)
Comment (Kavita539) [reply components/badge/badge.css]: Yeah thanks will be more specific

**Review by prathmeshjagtap — APPROVED** [review]

**Review by abhijeetekad — APPROVED** [review]

**Pull request merged**